Procfile

```diff
@@ -0,0 +1,2 @@
+web: ./docker/run-prod.sh
```

Comment:
Also these types of systems typically have special handling for the `web` type with ports and whatnot. We need to test all of this for sure. Perhaps I can push this to my demo to see what it does with an existing app?